.gitignore

```diff
@@ -2,6 +2,7 @@
 bin/
 # third-party dependencies via go mod
 vendor/
-
 # ginkgo test coverage files
 *.coverprofile
+# IntelliJ settings
```

Comment:
nit: when you merge in the conflict from jyoti's change, can you just lump these under `#IDE settings`?